### PR TITLE
Conditionally render language selection component

### PR DESCRIPTION
**Pull request by CharString**
Closes open-formulieren/open-forms#2254

Based on #273

### CODE REVIEW

File: src/components/App.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <BrowserRouter>
+      <App form={form} {...args} />
+    </BrowserRouter>
+  );
+};
+
+/*
+  FIXME: Why doesn't Storybook render these docs?
+  Uncaught TypeError: string is undefined
+*/
```
Comment (CharString) [inline src/components/App.stories.mdx:95]: Very disorienting to debug this... One thing I learned is that whitespace is significant.

**Comment by CharString** [comment]
May need another rebase on the branch attached to #273.

File: src/components/App.js [hunk 2]
```diff
@@ -2,22 +2,23 @@ import React from 'react';
...
 import Form from 'components/Form';
 import { Layout, LayoutRow } from 'components/Layout';
 import ManageAppointment from 'components/appointments/ManageAppointment';
 import LanguageSelection from 'components/LanguageSelection';
 
-
-const LanguageSwitcher = ({ target = null }) => (
-  target ? (
+const LanguageSwitcher = ({ target = null, renderSelector = false }) => {
+  if (!renderSelector) return (<></>);
```
Comment (sergei-maertens) [inline src/components/App.js:13]: `return null` is more idiomatic in react instead of using an empty fragment.

File: src/components/App.js [hunk 3]
```diff
@@ -29,7 +30,7 @@ Top level router - routing between an actual form or supporting screens.
 const App = ({ languageSelectorTarget, ...props }) => {
   return (
     <Layout>
-      <LanguageSwitcher target={languageSelectorTarget} />
+      <LanguageSwitcher target={languageSelectorTarget} renderSelector={props.form.translationEnabled} />
```
Comment (sergei-maertens) [inline src/components/App.js:33]: it makes a bit more sense to decide not to render a node in the parent (`App` in this case) if you only pass along that information:

```jsx
{props.form.translationEnabled ? <LanguageSwitcher ... /> : null}
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
I'll apply the changes and merge the PR

File: src/components/App.js [hunk 4]
```diff
@@ -50,6 +52,7 @@ const App = ({ languageSelectorTarget, ...props }) => {
 
 App.propTypes = {
   languageSelectorTarget: PropTypes.instanceOf(Element),
+  form: Types.Form,
```
Comment (sergei-maertens) [inline src/components/App.js:55]: we have some re-usable prop types defined in the `types` package :)

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**